README.md

```diff
@@ -1,20 +1,145 @@
-# suomifi-design-tokens
+# Suomifi Design System Tokens
```

Comment:
A short description of what the whole thing is about would be good here. Just an overall description of what the reader is looking at and what the tokens are for. Probably obvious for most people reading this, but still a good practice in my opinion.

Reply:
Added something, but we may need to consider how this repo is intended to be used and adjust accordingly.